tools/install_mac_deps.sh

```diff
@@ -5,30 +5,38 @@
 set -e  # Exit on error.
 set -x  # Make command execution verbose
 
-# Install some custom requirements on OS X using brew
-BREW=/usr/local/bin/brew
-if [[ ! -x $BREW ]]; then
-    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-fi
 
-BOOST_LIB="boost@1.76"
+# Set up Homebrew differently for arm64.
+if [[ $(uname -m) == 'arm64' ]]; then
```

Comment:
This probably should be guarded to check if brew is not installed (if we're going to suggest that this script could be run standalone and not only as a part of CI). I do not want the line to added each time I am running the script. Though I do not care as I am not using `zsh` :)